.env

```diff
@@ -0,0 +1,8 @@
+
+# Database Setup Enviroment File
+DB_ENV= "production"
+DATABASE_URL="postgres://fdzcjtkrxdgnqc:b96d68fefdfc9b5c834c867a0e5eb39d50f850127eebcf0fc367a81a0c720b5a@ec2-50-19-208-11.compute-1.amazonaws.com:5432/d1dm0flkjvtcmf"
+
+
+# JWT Token
+JWT_SECRET = AbCd1234!@
```

Comment:
I don't think it's best practice to include `.env` files in `git` repositories.

Reply:
This is interesting because git ignore has this in the file to be ignored as for why it got added I am still unsure. I Che ked it multiple times so not completely aware as to why